www/public/case-studies-rss.xml

```diff
@@ -0,0 +1,44 @@
+
+  <rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom">
+    <channel>
+      <title>Blog - Supabase</title>
+      <link>https://supabase.com</link>
+      <description>Latest news from Supabase</description>
+      <language>en</language>
+      <lastBuildDate>Invalid Date</lastBuildDate>
+      <atom:link href="https://supabase.com/blog/rss.xml" rel="self" type="application/rss+xml"/>
+      
+<item>
+  <guid>https://supabase.com/blog//case-studies/xendit</guid>
+  <title>Xendit</title>
+  <link>https://supabase.com/blog//case-studies/xendit</link>
+  <description>Two months of building</description>
+  <pubDate>Invalid Date</pubDate>
+</item>
+
+<item>
+  <guid>https://supabase.com/blog//case-studies/replenysh</guid>
+  <title>Replenysh</title>
+  <link>https://supabase.com/blog//case-studies/replenysh</link>
+  <description>Two months of building</description>
+  <pubDate>Invalid Date</pubDate>
+</item>
+
+<item>
+  <guid>https://supabase.com/blog//case-studies/mobbin</guid>
+  <title>Mobbin</title>
+  <link>https://supabase.com/blog//case-studies/mobbin</link>
+  <description>Two months of building</description>
+  <pubDate>Invalid Date</pubDate>
+</item>
+
+<item>
+  <guid>https://supabase.com/blog//case-studies/epsilon3</guid>
```

Comment:
Looks like these have double slashes in them @MildTomato `blog//` - is that right?

Reply:
thats wrong. this file doesn't need to be here at all actually. ill remove it from being generated

Reply:
removed